service/src/app/server.py

```
@@ -46,43 +50,161 @@ class LoadResponse(BaseModel):
 def load_raw(app_state: AppState, object_type: str, data: Any) -> LoadResponse:
+    handler = get_load_handler(object_type)
     items = data["items"]
-    return load_persons(app_state, items)
+    return handler(app_state, items)
+
+
+def get_load_handler(object_type: str) -> Callable[[AppState, list[Any]], LoadResponse]:
+    try:
+        return load_handlers[object_type]
+    except KeyError:
+        raise ClientException(detail="Unsupported data type")
```

Ruff (0.15.11)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)